app/src/main/res/values/strings.xml

```diff
@@ -25,10 +25,10 @@
     <string name="menu_filter">Filter</string>
     <string name="menu_clear">Clear completed</string>
     <string name="menu_delete_task">Delete task</string>
-    <string name="navigation_view_header_title">Todo</string>
+    <string name="navigation_view_header_title">Tasks</string>
     <string name="title_hint">Title</string>
     <string name="description_hint">Enter your task here.</string>
-    <string name="empty_task_message">TO DOs cannot be empty</string>
+    <string name="empty_task_message">Tasks cannot be empty</string>
```

Comment:
It's fine here I guess

Reply:
I'm keeping this - this is what started me on my todo purge - first of all it's TO DO instead of Todo - also they are referred to as tasks everywhere else (except the title of the app)